tuxemon/technique.py

```diff
@@ -238,35 +254,63 @@ def use(self, user: Monster, target: Monster) -> TechniqueResult:
             "capture": False,
             "statuses": [],
         }
-
         # TODO: handle conflicting values from multiple technique actions
         # TODO: for example, how to handle one saying success, and another not?
+       
         for effect in self.effect:
             if effect == "damage":
                 result = self.damage(user, target)
             elif effect == "poison":
                 result = self.apply_status("status_poison", target)
+            elif effect == "statchange":
+                result = self.changestats(user, target)
             elif effect == "lifeleech":
                 result = self.apply_lifeleech(user, target)
             elif effect == "recover":
                 result = self.apply_status("status_recover", user)
             elif effect == "status":
-                if self.category == "poison":
-                    result = self.poison(target)
-                elif self.category == "lifeleech":
-                    result = self.lifeleech(target)
-                elif self.category == "recover":
-                    result = self.recover(target)
-                else:
-                    result = getattr(self, self.category)(target)
+                for category in self.category:
+                    if category == "poison":
+                        result = self.poison(target)
+                    elif category == "lifeleech":
+                        result = self.lifeleech(target)
+                    elif category == "recover":
+                        result = self.recover(target)
+                    else:
+                        result = getattr(self, self.category)(target)
             else:
                 result = getattr(self, str(effect))(user, target)
             meta_result = merge_results(result, meta_result)
 
         self.next_use = self.recharge_length
 
         return meta_result
-
+    def changestats(self, user: Monster, target: Monster) -> EffectResult:
+        statsmaster = [self.statspeed, self.stathp, self.statarmour, self.statmelee, self.statranged, self.statdodge]
+        statslugs = ['speed', 'hp', 'armour', 'melee', 'ranged', 'dodge']
+        print(statsmaster)
+        if None not in statsmaster and [] not in statsmaster:
```

Comment:
This check is difficult to understand.  i think that you are looking for an empty list, or a list of Nones.   Consider rewriting it by iterating over the zippled list first, then checking for None or empty values.  It would be easier to understand what is going on.

Reply:
im unsure how to do that

Reply:
please don't resolve comments if there are no changes to address it.  try something like this:
```python
for stat, slug in zip(statsmaster, statslugs):
    if not stat:
        continue
```